src/services/about.rs

```diff
@@ -1,17 +1,30 @@
 extern crate reqwest;
 
 use crate::config::{base_transaction_service_url};
-use crate::models::backend::about::About;
+use crate::models::backend::about::About as AboutDto;
 use anyhow::Result;
 use reqwest::Url;
+use crate::models::service::about::About;
+use std::process::Command;
 
-pub fn get_about() -> Result<String> {
+pub fn get_about() -> Result<About> {
     let url_string = format!("{}{}", base_transaction_service_url(), "/about");
     let url = Url::parse(&url_string)?;
     let body = reqwest::blocking::get(url)?.text()?;
```

Comment:
I am not sure if I would include a service version. If we don't include it we can avoid the request making this endpoint a lot simpler.